src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
let's follow our [common mistakes](https://github.com/mate-academy/jv-oop/blob/master/checklist.md) and remove 4th line